### PR TITLE
metarelate date and git sha

### PR DESCRIPTION
**Pull request by marqh**
date and commit sha stamps for metarelate sourced content

as requested in #1929

### CODE REVIEW

File: lib/iris/fileformats/grib/_grib_cf_map.py [hunk 1]
```diff
@@ -16,6 +16,12 @@
 # along with Iris.  If not, see <http://www.gnu.org/licenses/>.
 #
 # DO NOT EDIT: AUTO-GENERATED
+# Created on 16 February 11 12:02 from
```
Comment (rhattersley) [inline lib/iris/fileformats/grib/_grib_cf_map.py]: Thanks for adding this @marqh :+1: :grin: ... just one small extra favour, presumably `16 February 11` => 11th of February 2016? Could you change the date format to be less ambiguous?

File: tools/gen_translations.py [hunk 2]
```diff
@@ -203,10 +210,11 @@ def build_grib_cf_map(fuseki, filename):
 
 
 def main():
+    now = datetime.utcnow().strftime('%y %B %d %H:%m')
```
Comment (rhattersley) [inline tools/gen_translations.py]: Aha ... this is the culprit!

**Comment by marqh** [comment]
> Could you change the date format to be less ambiguous?

eerk, sorry, that was somewhat unclear

i've altered this and added a check, just to protect against a race condition

**Comment by rhattersley** [comment]
Super - thanks @marqh . :+1:

**Pull request merged**